src/services/user.ts

```diff
@@ -0,0 +1,22 @@
+import { WithValidationError } from "../types/withValidationError"
+import { ApiCallOptions, deletePath, handleValidationError } from "../base"
+
+export const deleteUser = async ({
+  options = {},
+}: {
+  options?: ApiCallOptions
+}): Promise<WithValidationError> => {
+  try {
+    await deletePath({
+      path: `users/me`,
+      options: { isAuthorizedRequest: true, ...options },
+    })
+
+    return {
+      tag: "success",
+      result: {},
+    }
+  } catch (error) {
+    return handleValidationError(error)
```

Comment:
What kind of validation error are we getting from a delete endpoint?

Reply:
oh..actually we don't, so will re-add swallowErrors again

Reply:
I meant more like the handleValidationError function, in general, is very specific to how we handle form submissions that are having backend validations. In this case I would backend expect to only return status codes like 200/401 which are the only two cases we will ever have on deleting a user....

Reply:
I would still keep it for consistency, this is what we do in other places

Reply:
As described, I think the majority of times we do that is to handle form validation. I don't see a point in adding envelopes designed to handle and display validation errors if there is no case.